src/common/updating/updating.cpp

```diff
@@ -102,6 +108,11 @@ namespace updating
                 }
             }
 
+            if (blockNonPatchReleases && github_version >= VersionHelper{ 0, 36, 0 })
+            {
+                co_return version_up_to_date{};
```

Comment:
We should provide a different message, since in this case we want to notify the users that there is a new version but is not supported by their Windows < 1903.

Reply:
That function is only called by the periodic update checker (not counting Settings button, since it's impossible), so having a message here won't be displayed anywhere.

Reply:
Do we have any UI to tell the users that there is a new version but it's not supported on their system?

Reply:
we don't

Reply:
After further discussion, we decided to merge this PR as it is and open a new one to add a warning message to the old settings.